packages/react-components/src/stories/Tag.stories.tsx

```diff
@@ -0,0 +1,125 @@
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="info">
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="warning">
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="success">
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="error">
+          {children}
+        </TagComponent>
+      </div>
+      <div className="spacer" style={{ display: 'flex' }}>
+        <TagComponent {...args} kind="default" outline>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="info" outline>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="warning" outline>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="success" outline>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="error" outline>
+          {children}
+        </TagComponent>
+      </div>
+      <div className="spacer" style={{ display: 'flex' }}>
+        <TagComponent {...args} kind="default" icon={Smiles}>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="info" icon={Smiles}>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="warning" icon={Smiles}>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="success" icon={Smiles}>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="error" icon={Smiles}>
+          {children}
+        </TagComponent>
+      </div>
+      <div className="spacer" style={{ display: 'flex' }}>
+        <TagComponent {...args} kind="default" avatar={avatar}>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="info" avatar={avatar}>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="warning" avatar={avatar}>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="success" avatar={avatar}>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="error" avatar={avatar}>
+          {children}
+        </TagComponent>
+      </div>
+      <div className="spacer" style={{ display: 'flex' }}>
+        <TagComponent {...args} kind="default" icon={Smiles} dismissible>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="info" icon={Smiles} dismissible>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="warning" icon={Smiles} dismissible>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="success" icon={Smiles} dismissible>
+          {children}
+        </TagComponent>
+        <TagComponent {...args} kind="error" icon={Smiles} dismissible>
+          {children}
+        </TagComponent>
+      </div>
```

Comment:
`dismissable` is just an option that could be configured in the first story. I think we don't need to list all kinds only for showing `X` icon.